src/components/features/Forest/TreeList/index.tsx

```diff
@@ -4,16 +4,23 @@ import TreeItemAdd from './TreeItemAdd';
 import * as S from './TreeList.styled';
 import { TREE_SIZE_MAX } from '@/constants/forest';
 import { Props } from './TreeList.type';
+import { useParams } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { myInfoState } from '@/stores/user';
 
 const TreeList = ({ trees }: Props) => {
+	const myInfo = useRecoilValue(myInfoState);
+	const { userId: currentforestUserId } = useParams();
+
```

Comment:
```suggestion
	const myInfo = useRecoilValue(myInfoState);
	const { userId: currentForestUserId } = useParams();

```
중요한건 아니지만 오타인 것 같습니다!